### PR TITLE
bump cryptography above 41.0.3

### LINKED ISSUE #19: cryptography<41.0.3 includes vulnerable OpenSSL version
see [Vulnerable OpenSSL included in cryptography wheels](https://github.com/advisories/GHSA-5cpq-8wj7-hf2v) and [pyca/cryptography's wheels include vulnerable OpenSSL](https://github.com/advisories/GHSA-jm77-qphf-c4w8)

python-dotenv-vault [depends on](https://github.com/dotenv-org/python-dotenv-vault/blob/master/requirements.txt) `cryptography<41.0.0,>=3.1.0` which prevents package managers resolving a higher version e.g.:
```
(venv) @mnbf9rca ➜ /workspace (chore/update_deps) $ poetry add cryptography@>=41.0.3

Because python-dotenv-vault (0.6.3) depends on cryptography (>=3.1.0,<41.0.0)
 and no versions of python-dotenv-vault match >0.6.3,<0.7.0, python-dotenv-vault (>=0.6.3,<0.7.0) requires cryptography (>=3.1.0,<41.0.0).
So, because mqtt-to-eventhub depends on both python-dotenv-vault (^0.6.3) and cryptography (^41.0.3), version solving failed.
```

### PR DESCRIPTION
**Pull request by mnbf9rca**
resolves #19 
adds additional tests (load_dotenv() is now tested)

### CODE REVIEW

**Review by nsnguyen — APPROVED** [review]
this lgtm, thanks @mnbf9rca !

**Comment by mnbf9rca** [comment]
@nsnguyen i can't merge this PR as i dont have write access to the repo - would you mind merging it?

**Comment by nsnguyen** [comment]
it's merged now @mnbf9rca . Thanks 👍

**Pull request merged**
